xcube/core/imgeom.py

```diff
@@ -277,6 +277,11 @@ def _compute_output_geom(dataset: xr.Dataset,
     dst_height = 1 + math.floor((src_y_max - src_y_min) / src_xy_res)
```

Comment:
```suggestion
    dst_height = math.round((src_y_max - src_y_min) / src_xy_res) or 1
```

Reply:
See [my comment](https://github.com/dcs4cop/xcube/issues/303#issuecomment-638114126) on the issue. The preferred approach depends on the desired behaviour: `1 + math.floor...` will always provide enough pixels to cover the desired area. `math.round` will sometimes round down so that not all the desired area is covered -- but since the missing cropped area is less than a pixel-width, this is probably just nitpicking.

Reply:
Ok, agree!